transforms/angle-brackets/test.js

```diff
@@ -109,29 +109,29 @@ test('data-attributes', () => {
 
   expect(runTest('data-attributes.hbs', input)).toMatchInlineSnapshot(`
     "
-        <XFoo data-foo={{true}} />
-        <XFoo data-test-selector={{true}} />
-        <XFoo data-test-selector={{post.id}} />
-        <XFoo @label=\\"hi\\" data-test-selector={{true}} />
-        <XFoo data-test-foo />
+        <XFoo @data-foo={{true}} />
+        <XFoo @data-test-selector={{true}} />
+        <XFoo @data-test-selector={{post.id}} />
+        <XFoo @label=\\"hi\\" @data-test-selector={{true}} />
+        <XFoo @data-test-foo />
```

Comment:
I don't think this is valid 🤔

Reply:
probably best to skip this particular conversion and keep it as a curlies

Reply:
All of those are invalid or just one of them?

Reply:
Its very easy fix for data-less to be `data-foo` instead of `@data-foo`

Reply:
just this particular one because it would result in an angle bracket component with a positional argument ;)

Reply:
Let me see if I can fix that easily.

Reply:
Which one of these is legal?
```hbs
<FooBar @data-baz @class="baz" />
```
or
```hbs
<FooBar data-baz @class="baz" />
```

Reply:
the second one would be legal, but only if `FooBar` correctly uses `...attributes`. I think we should completely skip the conversion and keep it as `{{foo-bar data.baz class="baz"}}` instead.

Reply:
Skip all data attribute cases or specific cases?  Just give me the exact details of what you want and I’ll wrap this up. 😀

Reply:
skip conversion for components with `data-foo`, but convert `data-foo="bar"` to `@data-foo="bar"`. for built-in components we can convert `data-foo` to `data-foo`.

Reply:
You got it dude 🎉. I’ll do it once I get up!